### PR TITLE
Updating Angor Docs

### PR DESCRIPTION
**Pull request by Danielogenna**

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -19,8 +19,36 @@ export default defineConfig({
           autogenerate: { directory: 'overview' },
         },
         {
-          label: 'How to Invest using Angor',
-          autogenerate: { directory: 'how-to-investsing Angor' },
+          label: 'getting started',
+          autogenerate: { directory: 'creating a wallet' },
```
Comment (miladsoft) [inline astro.config.mjs:23]: Use - instead of space for directory name

File: src/content/docs/creating-a-project/creating-a-project.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Creating a Project
+
+1. Sign In: Log in to your Angor account.
```
Comment (miladsoft) [inline src/content/docs/creating-a-project/creating-a-project.md:3]: We don't have login to the account and only the wallet is created or recovered

File: astro.config.mjs (hunk 1 above)
Comment (Danielogenna) [reply astro.config.mjs:23]: Noted

File: src/content/docs/creating-a-project/creating-a-project.md (hunk 2 above)
Comment (Danielogenna) [reply src/content/docs/creating-a-project/creating-a-project.md:3]: Oh yeah I missed this out, will make adjustments.

**Comment by miladsoft** [comment]
not yet

**Comment by Danielogenna** [comment]
@miladsoft corrected all changes now, can check.

**Pull request merged**